### PR TITLE
[MOBILE APP] codebase cleanup

### PR DESCRIPTION
**Pull request by NoahNsimbe**
#### Summary of Changes (What does this PR do?)

- [x] Updates packages

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [ ] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

### CODE REVIEW

**Review by BenjaminSsempala — APPROVED** [review]
LGTM

File: mobile/README.md [hunk 1]
```diff
@@ -138,6 +138,8 @@ flutter pub run dart_code_metrics:metrics check-unused-code lib
 ### **Building for release**
 
 ```bash
+flutter build appbundle --build-name 1.0.0 --build-number 20025 --flavor airqo
```
Comment (BenjaminSsempala) [inline mobile/README.md:141]: @NoahNsimbe do we have to change this build number in the README every time we update

Comment (NoahNsimbe) [reply mobile/README.md:141]: No. the README is just or reference. We can explore leveraging the above command to build and tag the app with a version without modifying the version in the pubspec file.

File: mobile/pubspec.lock [hunk 2]
```diff
@@ -61,10 +61,10 @@ packages:
     dependency: transitive
     description:
       name: archive
-      sha256: ed7cc591a948744994714375caf9a2ce89e1d82e8243997c8a2994d57181c212
+      sha256: d6347d54a2d8028e0437e3c099f66fdb8ae02c4720c1e7534c9f24c10351f85d
```
Comment (Mnoble-19) [inline mobile/pubspec.lock:64]: @NoahNsimbe, I've made these changes `flutter pub upgrade` because the build was originally failing.

Comment (NoahNsimbe) [reply mobile/pubspec.lock:64]: No worries @Mnoble-19.  This is fine. It won't break anything. LGTM

**Pull request merged**